specification/cognitiveservices/data-plane/Language/readme.md

```diff
@@ -6,9 +6,21 @@ This is the AutoRest configuration file the Cognitive Services Language SDK.
 
 ## Releases
 
+The current stable release of Question Answering is 2021-10-01.
+
+The current stable release of Language is 2022-05-01.
+
 The current preview release of Conversational Language Understanding and Text Analytics is 2022-03-01-preview.
 
-The current stable release of Question Answering is 2021-10-01.
+```yaml
+tag: release_2022_05_01
+add-credentials: true
+clear-output-folder: true
+openapi-type: data-plane
+directive:
+  - suppress: LongRunningResponseStatusCode
+    reason: The validation tools do not properly recognize 202 as a supported response code.
+```
```

Comment:
There can only be one of these sections. Please delete the others: with your changes in place, that would be lines 25 through 68 inclusively.

These are global tags. Repeating them changes behaviors. Not sure if tools are first- or last-wins, but having only a single will make that moot.

Reply:
removed